nixos/server/default.nix

```diff
@@ -17,6 +17,8 @@
     pkgs.vim
   ];
 
+  documentation.enable = lib.mkDefault false;
```

Comment:
```suggestion
  # Notice this also disables --help for some commands such es nixos-rebuild
  documentation.enable = lib.mkDefault false;
```

Reply:
I had the idea to turn this into a standalone package:

```
$ nix repl
Welcome to Nix 2.13.3. Type :? for help.

nix-repl> :lf .#                                                         
Added 19 variables.

nix-repl> :b nixosConfigurations.eve.config.system.build.manual.manpages

This derivation produced the following outputs:
  out -> /nix/store/919zbm49lcizzdd66myyrgwzkb8fm444-nixos-manpages

nix-repl> 
```

similar to nix-index-database.

Reply:
I started some repo, will update you once I got it running.